package.json

```diff
@@ -11,104 +13,27 @@
-        "codegen",
-        "yaml",
-        "json",
-        "fetch",
-        "xhr",
-        "axios",
-        "angular",
-        "node"
-    ],
-    "main": "./dist/node/index.js",
-    "types": "./dist/node/index.d.ts",
-    "bin": {
-        "openapi-ts": "bin/index.js"
-    },
-    "files": [
-        "bin",
-        "dist"
-    ],
     "scripts": {
-        "build-bundle": "rollup --config rollup.config.ts --configPlugin typescript",
-        "build-types-check": "tsc --project tsconfig.check.json",
-        "build-types-roll": "rollup --config rollup.dts.config.ts --configPlugin typescript && rimraf temp",
-        "build-types-temp": "tsc --emitDeclarationOnly --outDir temp -p src/node",
-        "build-types": "run-s build-types-temp build-types-roll build-types-check",
-        "build": "run-s clean build-bundle build-types",
-        "clean": "rimraf dist test/generated test/e2e/generated coverage node_modules/.cache",
-        "dev": "rimraf dist && npm run build-bundle -- --watch",
-        "lint:fix": "eslint . --fix",
-        "lint": "eslint .",
-        "prepublishOnly": "npm run build",
-        "test:coverage": "vitest run --config vitest.config.unit.ts --coverage",
-        "test:e2e": "vitest run --config vitest.config.e2e.ts",
-        "test:sample": "node test/sample.cjs",
-        "test:update": "vitest watch --config vitest.config.unit.ts --update",
-        "test:watch": "vitest watch --config vitest.config.unit.ts",
-        "test": "vitest run --config vitest.config.unit.ts",
-        "typecheck": "tsc --noEmit",
+        "build": "npm run build --workspaces --if-present",
+        "lint:fix": "npm run lint:fix --workspaces --if-present",
+        "lint": "npm run lint --workspaces --if-present",
+        "test:coverage": "npm run test:coverage --workspaces --if-present",
+        "test:e2e": "npm run test:e2e --workspaces --if-present",
+        "test:update": "npm run test:update --workspaces --if-present",
+        "test": "npm run test --workspaces --if-present",
+        "typecheck": "npm run typecheck --workspaces --if-present",
+        "openapi-ts": "npm run --workspace packages/openapi-ts --",
         "changeset": "changeset"
     },
     "engines": {
         "node": "^18.0.0 || >=20.0.0"
     },
-    "dependencies": {
-        "@apidevtools/json-schema-ref-parser": "11.5.4",
-        "camelcase": "8.0.0",
-        "commander": "12.0.0",
-        "handlebars": "4.7.8"
-    },
     "devDependencies": {
-        "@angular-devkit/build-angular": "17.3.2",
-        "@angular/animations": "17.3.2",
-        "@angular/cli": "17.3.2",
-        "@angular/common": "17.3.2",
-        "@angular/compiler": "17.3.2",
-        "@angular/compiler-cli": "17.3.2",
-        "@angular/core": "17.3.2",
-        "@angular/forms": "17.3.2",
-        "@angular/platform-browser": "17.3.2",
-        "@angular/platform-browser-dynamic": "17.3.2",
-        "@angular/router": "17.3.2",
         "@changesets/cli": "2.27.1",
```

Comment:
Changesets stays in workspace root. and will eventually allow you to select which packages to version (it handles monorepos automatically, you will see once we have more packages)